packages/react-components/src/entities/SiteSidebar/SiteSidebar.js

```diff
@@ -96,17 +96,19 @@ export function SiteSidebar({
 
 export function TemporalDisplay({ year, month }) {
   if (year && month && month > 0) {
+
+    let monthFormatted = useIntl().formatMessage({ id: `enums.month.${month}` });
     return (
       <Eyebrow 
         style={{fontSize: '80%'}}
-        prefix={<>Details of activity in <FormattedMessage id={`enums.month.${month}`} defaultMessage={`${month}`} />, {year}</>}
+        prefix={<><FormattedMessage id="eventDetails.activityDetails" values={{ date: monthFormatted + ', ' + year }}  /></>}
```

Comment:
Using intl date formatting seems a more natural fit?

Reply:
changed to use  intl date formatting